packages/carbon-web-components/src/globals/CarbonBase.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// @ts-nocheck
+import { LitElement } from 'lit';
+import CarbonStyles from '../internal/vendor/@carbon/ibmdotcom-utilities/utilities/CarbonStyles/CarbonStyles';
```

Comment:
Something isn't right about this import and it's causing `yarn start` within `packages/storybook` to fail.

I think it should look closer to the following the way the index.js / folder structures are, this is working for me:

I've been running `yarn build` at the project root, then running `yarn start` within `src/packages` to test. The following clears me of errors.

```suggestion
import { CarbonStyles } from '../internal/vendor/@carbon/ibmdotcom-utilities/utilities/carbonStyles/carbonStyles.js';
```